functional-wheel-launcher/lib/Utils/Logging-test.pm

```diff
@@ -0,0 +1,41 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+=head1 Utils::Logging-test
+
+C<Utils::Logging_test> - Save logs directly on the worker for offline upload via ulogs 
+
+=cut
+
+package Utils::Logging_test;
+
+use base 'Exporter';
+use Exporter;
+use strict;
+use warnings;
+use testapi 'script_output';
```

Comment:
I missed this during last review, this is not needed anymore.

Reply:
yes, the file has been stored in wheel repository